tests/robustness/traffic.go

```diff
@@ -64,26 +64,39 @@ func simulateTraffic(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2
 	limiter := rate.NewLimiter(rate.Limit(config.maximalQPS), 200)
 
 	startTime := time.Now()
+	c, err := NewClient(endpoints, ids, startTime)
```

Comment:
Minor comment: It isn't a problem to define the same variable name (line 67 and line 75), but please give it another name, just in case the following contributors accidentally change the `:=` to `=` at line 75.
```suggestion
	cc, err := NewClient(endpoints, ids, startTime)
```

Reply:
Done